VEDA_OS01/fixtures/encodes.json

```diff
@@ -160,5 +160,18 @@
             "encode_resolution": "720",
             "product_spec": "desktop_webm"
         }
+    },
+    {
+        "model": "VEDA_OS01.course",
+        "pk": 1,
+        "fields": {
+            "course_name": "Demo Course",
+            "institution": "XXX",
+            "edx_classid": "XXXXX",
```

Comment:
#NIT instead of `xxxxx`, can these fake value be something easier to read?

Reply:
This should be an 3/5 letter code that gets concatenated to (for example) MIT6002X, so it shouldn't be super easy to read, and ideally won't collide with another course (if added).
I'll do EDXDEMOX.